src/simplewallet/simplewallet.cpp

```diff
@@ -1579,12 +1578,12 @@ bool simple_wallet::start_mining(const std::vector<std::string>& args)
   }
   else
   {
-    ok = false;
+    req.threads_count = 1;
   }
 
   if (!ok)
   {
-    fail_msg_writer() << tr("invalid arguments. Please use start_mining [<number_of_threads>], "
+    fail_msg_writer() << tr("invalid arguments. Please use start_mining [<number_of_threads>] [do_bg_mining] [ignore_battery], "
```

Comment:
That's a bit confusing. Given that help, it's not clear you need to put in true or nothing.